src/simulator/Simulator.tsx

```diff
@@ -67,16 +74,20 @@ const Simulator = ({
   const simHeight = contentRect?.height ?? 0;
   const [brand500] = useToken("colors", ["brand.500"]);
   const [running, setRunning] = useState<RunningStatus>(RunningStatus.STOPPED);
+  const previouslyShown = usePrevious(shown);
 
   useEffect(() => {
     if (shown) {
-      ref.current!.focus();
+      // Prevents the simulator stealing focus on initial load.
```

Comment:
I'd really love to find a way to move away from effects for this but I think this is a reasonable fix at this point.